src/taskHandler.ts

```diff
@@ -37,23 +36,23 @@ export class TaskHandler {
     );
   }
 
-  public async waitForTask<T>(taskType: string): Promise<ITaskResponse<T>> {
+  public async waitForTask<T>(taskType: string, jobType: string): Promise<ITaskResponse<T>> {
```

Comment:
please put the jobType before the task type

Reply:
done